### ISSUE TITLE
Termux

### ISSUE THREAD

**Issue by sheeepdev** [13-CF/afetch#49]
There is no info nor support for termux. Could you please add support for termux?

**Comment by 13-CF** [comment]
Is there a `/etc/os-release` file?

**Comment by sheeepdev** [comment]
> 
> 
> Is there a `/etc/os-release` file?

There isn't

**Comment by sheeepdev** [comment]
but i have seen other fetches fetch info for termux.. so wouldn't it be possible?

**Comment by 13-CF** [comment]
Yeah but I need a way to find the OS without an `os-release`, similarly to how I do it on BSD.

What's the output of `uname -a`?

**Comment by sheeepdev** [comment]
Linux Localhost 4.14.117-19828683 #1 SMP PREEMPT FRI Nov 13 18:53:37 KST 2020 aarch64 Android

**Comment by 13-CF** [comment]
Does `uname -s` output `Linux` or `Android`?

**Comment by sheeepdev** [comment]
> 
> 
> Does `uname -s` output `Linux` or `Android`?

Linux

**Comment by SirCipherz** [comment]
according to neofetch code the presence of the directories "/system/app" and "/system/priv-app" can be used to indentify an android system

**Comment by 13-CF** [comment]
Hi there, sorry for the delay. Is there any files in `/system/app` and `/system/priv-app` that are on all installations? I know how to implement it but I don't have an Android system to test it on.

**Comment by sheeepdev** [comment]
> Hi there, sorry for the delay. Is there any files in `/system/app` and `/system/priv-app` that are on all installations? I know how to implement it but I don't have an Android system to test it on.

Im sorry for the very late response, it looks like the folders in ``/system/app`` and ``/system/priv-app`` are just system applications and have nothing to do with termux and have no way getting the information about the installation.
